frontend/src/container/LogsExplorerViews/index.tsx

```diff
@@ -578,6 +597,11 @@ function LogsExplorerViews({
 					<div className="logs-actions-container">
 						{selectedPanelType === PANEL_TYPES.LIST && (
 							<div className="tab-options">
+								<Download
+									data={flattenLogData}
+									isLoading={isLoading || isFetching}
```

Comment:
We can remove `isLoading` as `isFetching` will also be true in case `isLoading` is true. 

<img width="1039" alt="Screenshot 2024-03-21 at 1 27 37 PM" src="https://github.com/SigNoz/signoz/assets/26198062/6e296f56-8842-47ab-8ee5-cb6a565da96c">

Reply:
done